server/py/services/api/migrations/versions/fa304088c8fb_update_model_endpoints_table_2_0.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Update model endpoints table 2.0
```

Comment:
IIUC model endpoint table was introduced in 1.8 so you can remove all model endpoints migrations and create a new one with the latest table schema